libs/ui/src/lib/fire-calculator/fire-calculator.component.ts

```diff
@@ -86,7 +91,8 @@ export class FireCalculatorComponent
         annualInterestRate: 5,
         paymentPerPeriod: this.savingsRate,
         principalInvestmentAmount: 0,
-        time: 10
+        time: 10,
+        retirementNetWorth: this.targetNetWorth
```

Comment:
Please move up (for an alphabetical order)

Reply:
Done